src/include/hwinfo/system_settings_ui.rb

```diff
@@ -58,7 +58,9 @@ def ReadSystemSettingsDialog
       # to avoid of the very long starting time of the yast module
       # because the Storage module (which is imported by the Bootloader (imported by the SystemSettings module))
       # has a Read() function call in its constructor.
-      SystemSettings.Read
+      system_settings = SystemSettings.Read
```

Comment:
This could be write

```ruby
return :abort unless SystemSettings.Read
```

but sometimes a received the feedback that hidden important things after conditional statements is not so good.

So, I leave the decision in the code reviewer ;)

Reply:
I personally prefer the version in the comment, but i have no strong preference. A matter of taste, I guess.